tests/integration/test_hyper_transformer.py

```diff
@@ -293,7 +293,7 @@ def test_default_inputs(self):
                     0.3024284729840169,
                 ],
                 'datetime': [
-                    1.2630692571428572e18,
+                    1.2631629169758298e18,
```

Comment:
I believe this changed because the `NaN` replacement for datetime is now `random`, which also changed the below `reversed_datetimes`